applications/ecallmgr/src/ecallmgr_call_command.erl

```diff
@@ -1275,10 +1275,12 @@ maybe_get_terminators(Acc, JObj) ->
 -spec start_record_call_args(atom(), ne_binary(), wh_json:object(), ne_binary()) -> ne_binary().
 start_record_call_args(Node, UUID, JObj, RecordingName) ->
     FollowTransfer = wh_json:get_binary_boolean(<<"Follow-Transfer">>, JObj, <<"true">>),
+    RecordMinSec = wh_json:get_value(<<"Record-Min-Sec">>, JObj),
```

Comment:
This should probably be `get_binary_value/2` to ensure the integer is encoded as a binary and not skipped in `ecallmgr_util:set/3`.